bin/cli.js

```diff
@@ -13,12 +14,15 @@ const teamActions = require('../src/team.actions');
 const teamsActions = require('../src/teams.actions');
 
 const opn = require('opn');
-const argv = require('yargs')
+const yargs = require('yargs')
```

Comment:
.. and bonus!  
I feel more like a pirate saying this too! ☠️ ⚓️ ⛵️

Reply:
That's really the main benefit of this PR :)